### PR TITLE
sander/PrepV2Release

### PR DESCRIPTION
**Pull request by SanderElias**
- Added to FAQ page
- chore(monorepo): update to latest NX && Angular
- chore(monorepo): update deps to latest versions
- docs(docsWeb): add roadmap, consultancy and form for adding links.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/scullyio/scully/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Other... Please describe:

This PR is the base for the V2 release

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: apps/scully-docs/src/app/app.component.ts [hunk 1]
```diff
@@ -26,6 +26,7 @@ import { NavListService } from './components/nav-list/nav-list.service';
 export class AppComponent {
   constructor(private router: Router, public nl: NavListService) {}
   get isLandingPage() {
-    return this.router.url === '/' || this.router.url === '/ngconf';
+    return !this.router.url.toLowerCase().includes('/docs')
+    // return this.router.url === '/' || this.router.url === '/ngconf';
```
Comment (Jefiozie) [inline apps/scully-docs/src/app/app.component.ts:30]:
```suggestion
    return !this.router.url.toLowerCase().includes('/docs')
```

File: docs/consutancy.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: Scully consultancy.
+published: true
+lang: en
+position: 2000
+---
+
+# Scully consultancy.
+
+If you have questions, or want to have us help you intergate Scully in your current project, contact the team at HeroDevs.
```
Comment (Jefiozie) [inline docs/consutancy.md:10]:
```suggestion
If you have questions or want to have us help you integrate Scully into your current project, contact the team at HeroDevs.
```

File: docs/faq.md [hunk 3]
```diff
@@ -222,6 +222,49 @@ Also, make sure you add the following to your config:
 
 </details>
 
+<details>
+<summary>Scully inside GCE has timeout failures.</summary>
+
+It seems that inside GCE sometimes the server takes a longer time to properly come up. If this happens, you can extend the waiting time for the server with a command line parameter like:
```
Comment (Jefiozie) [inline docs/faq.md:228]:
```suggestion
It seems that inside GCE sometimes the server takes a long time to properly come up. If this happens, you can extend the waiting time for the server with a command-line parameter like:
```

File: docs/faq.md [hunk 4]
```diff
@@ -222,6 +222,49 @@ Also, make sure you add the following to your config:
...
+
+```bash
+npx scully --handle404=index --hostName="${SSR_HOST_NAME}" --noPrompt  --serverTimeout=60000
+```
+Also the following puppeteer settings are reported to help in this case:
```
Comment (Jefiozie) [inline docs/faq.md:233]:
```suggestion
The following puppeteer settings are reported to help in this case:
```

**Review by Jefiozie — APPROVED** [review]
Minor comments, LGTM!

**Review by Jefiozie — APPROVED** [review]

File: docs/roadmap.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+- release V2 of Scully
+  - Deprecate support for Angular before version 12.
+  - V2 will require Angular v12 and up
+  - V2 will switch over to the new Ivy library structure soon
+  - V1 will remain available for Angular versions 9/10/11
+  - support for versions before V9 is completely dropped.
+- Better control over server-start-stop.
+  - We have had some reports of stalling in CI/CD, an this seems to be the root cause of this issue.
+- Renderer build on top of [platform-server](https://angular.io/api/platform-server)
```
Comment (aaronfrost) [inline docs/roadmap.md:21]: Add "Optionally" since some of the AngularJS clients might freak.

File: docs/roadmap.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+  - Scully is now using Puppeteer to render the resulting pages. This approach will be using the platform server. While this puts some additional demands on the application (similar to Angular Universal, which uses the same engine), it can render pages _much_ faster. (It doesn't have to start a browser, navigate to a page and so on)
```
Comment (aaronfrost) [inline docs/roadmap.md:22]: Scully will still default to render with Puppeteer. Turning on Platform-server is opt-in.

**Review by Jefiozie — APPROVED** [review]
LGTM

**Pull request merged**